src/bulk/common.ts

```diff
@@ -334,9 +332,9 @@ export class BulkWriteResult {
  * @category Error
  */
 export class WriteConcernError {
-  err: MongoServerError;
+  err: { code: number; errmsg: string; errInfo?: Document };
```

Comment:
We technically have a breakage here, this is no longer `instanceof MongoServerError` but this was just relying on the key mapping logic of the MongoServerError constructor. 
I think its fair to make this change, all the properties are still accessible. Of course, up for discussion:

Reply:
You want to map these props on to this class? I think that sounds good removes a layer of indirection (and we keep a getter to prevent breakage)

Reply:
something along the lines of defining a `kErr` symbol (maybe more descriptive) set equal to the `{ code, errmsg, errInfo }` and then a public getter marked deprecated: `get err() { return this[kErr] }` so that we hopefully don't break any existing functionality in case someone is trying to read directly from the current err prop